test/JDBC/expected/alter-procedure-vu-prepare.out

```diff
@@ -22,6 +22,9 @@ go
 create procedure alter_proc_p3 as select 1
 go
 
+create procedure alter_proc_p4 as select 1
```

Comment:
I don't see a test case where we do `alter procedure` in a prepare script and verify it in a verify script. Please add this case, if there's no such test already.